apps/admin-gui/src/app/shared/components/dialogs/assign-service-to-resource-dialog/assign-service-to-resource-dialog.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="{{theme}}">
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/88302537-40ddd880-cd06-11ea-9ae4-2ff2604b616c.png)
this dialog is too big on my notebook. Please look on the other dialogs how to add dialog if the height of the dialog is too big for viewport.